easybuild/easyconfigs/p/Pmw/Pmw-1.3.3-goolf-1.4.10-Python-2.7.5.eb

```diff
@@ -0,0 +1,37 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Adam Mazur
+# Research IT
+# Biozentrum, University of Basel
+
+easyblock = "PythonPackage"
+
+name = 'Pmw'
+version = '1.3.3'
+
+homepage = 'http://pmw.sourceforge.net'
+description = """Pmw is a toolkit for building high-level compound widgets in Python
+ using the Tkinter module. """
+
+toolchain = {'name': 'goolf', 'version': '1.4.10'}
+
+source_urls = ['http://sourceforge.net/projects/%(namelower)s/files/%(name)s/%(name)s.%(version)s/', '/download']
```

Comment:
the `/download` part is not needed?

as it is now, EasyBuild will try to download the source file from `/download/Pmw.1.3.3.tar.gz` in case the first source URL fails...